### PR TITLE
EORG bans

### PR DESCRIPTION
**Pull request by Gandalf2k15**
## About The Pull Request

Adds an EORG ban system.

## Changelog
:cl:
add: You can now be banned from EORG, do not EORG prep.
/:cl:

### CODE REVIEW

**Comment by lyricalpaws** [comment]
yes please

File: modular_skyrat/modules/banning/code/skyrat_bans.dm [hunk 1]
```diff
@@ -10,3 +12,29 @@
...
+
+/obj/effect/landmark/hell
+	name = "Hell"
+	icon_state = "portal_exit"
+
+/obj/effect/landmark/hell/Initialize(mapload)
+	..()
+	GLOB.hell += loc
+	return INITIALIZE_HINT_QDEL
+/obj/effect/landmark/holding_facility
```
Comment (Ghommie) [inline modular_skyrat/modules/banning/code/skyrat_bans.dm]: copypasta leftover.

Comment (Gandalf2k15) [reply modular_skyrat/modules/banning/code/skyrat_bans.dm]: oopsie

File: code/modules/shuttle/emergency.dm [hunk 2]
```diff
@@ -551,6 +551,7 @@
 
 				dock_id(destination_dock)
 				unbolt_all_doors() //SKYRAT EDIT ADDITION
+				INVOKE_ASYNC(src, .proc/process_eorg_bans) //SKYRAT EDIT ADDITION
```
Comment (Ghommie) [inline code/modules/shuttle/emergency.dm]:
```suggestion
				INVOKE_ASYNC(GLOBAL_PROC, .proc/process_eorg_bans) //SKYRAT EDIT ADDITION
```

**Comment by AbsoluteRawGamer** [comment]
Get robust

**Pull request merged**